docker-initunlocklnd.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
make sure it is correctly chmod'ed

`git update-index --chmod=+x docker-initunlocklnd.sh`

Reply:
ACK